ultrasonic/src/main/kotlin/org/moire/ultrasonic/adapters/TrackViewBinder.kt

```diff
@@ -89,6 +96,13 @@ class TrackViewBinder(
             }
         }
 
+        holder.drag.setOnTouchListener { _, event ->
```

Comment:
ItemTouchHelper can use a drag handle, and in some samples they do it like this.